src/usbhost/usbhost.c

```diff
@@ -676,7 +676,34 @@ int usb_device_bulk_transfer(struct usb_device *device,
     ctrl.len = length;
     ctrl.data = buffer;
     ctrl.timeout = timeout;
-    return ioctl(device->fd, USBDEVFS_BULK, &ctrl);
+
+    #if defined(__ANDROID__) && (__ANDROID_API__ < __ANDROID_API_P__)
```

Comment:
Rather than repeat this test, how about just using this?
```C++
#ifdef MAX_USBFS_BUFFER_SIZE
```

Reply:
This will work and reminds me about why code reviews are important ;)